bait-files/src/main/java/com/google/sps/servlets/Review.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+
+@Entity
+public class Review {
+  @Id Long reviewId;
+  String userId;
+  String requestId;
+  String status;
+  String nameValidity;
+  String usernameValidity;
+  String emailValidity;
+  String phoneNumValidity;
+  String addressValidity;
+  String imageValidity;
```

Comment:
I thought these were bools? What are these? Maybe an enum here as well then.

Reply:
Changed them to boolean